src/features/world/ui/deliveries/DeliveryModal.tsx

```diff
@@ -0,0 +1,234 @@
+import React, { useContext, useEffect, useState } from "react";
+import { Context } from "features/game/GameProvider";
+import { NPCName, NPC_WEARABLES } from "lib/npcs";
+import { MachineState } from "features/game/lib/gameMachine";
+import { useSelector } from "@xstate/react";
+import { SpeakingModal } from "features/game/components/SpeakingModal";
+import { getKeys } from "features/game/types/craftables";
+import { RequirementLabel } from "components/ui/RequirementsLabel";
+import Decimal from "decimal.js-light";
+import { defaultDialogue, npcDialogues } from "./dialogues";
+import { Inventory, Order } from "features/game/types/game";
+import { OuterPanel } from "components/ui/Panel";
+import { PIXEL_SCALE } from "features/game/lib/constants";
+
+import selectBoxBL from "assets/ui/select/selectbox_bl.png";
+import selectBoxBR from "assets/ui/select/selectbox_br.png";
+import selectBoxTL from "assets/ui/select/selectbox_tl.png";
+import selectBoxTR from "assets/ui/select/selectbox_tr.png";
+import { useRandomItem } from "lib/utils/hooks/useRandomItem";
+import classNames from "classnames";
+
+const OrderCards: React.FC<{
```

Comment:
<img width="545" alt="Screenshot 2023-07-20 at 4 37 57 pm" src="https://github.com/sunflower-land/sunflower-land/assets/11745561/f4721c81-31a9-488b-8767-65527b10584c">

Can we also display the reward that a player will receive on these cards? I feel this context would be helpful to reassure their action